.circleci/config.yml

```diff
@@ -108,19 +108,21 @@ jobs:
               set -e
             }
 
+            export LOCAL_IMAGE="local/socorro_app:latest"
+
             if [ "${DOCKER_USER}" == "" ] || [ "${DOCKER_PASS}" == "" ]; then
               echo "Skipping Login to Dockerhub, credentials not available."
             else
               echo "${DOCKER_PASS}" | docker login -u="${DOCKER_USER}" --password-stdin
 
               if [ "${CIRCLE_BRANCH}" == "main" ]; then
                 # deploy main latest
-                docker tag "local/socorro_app:latest" "${DOCKERHUB_REPO}:latest"
+                docker tag "${LOCAL_IMAGE}" "${DOCKERHUB_REPO}:latest"
                 retry docker push "${DOCKERHUB_REPO}:latest"
               elif  [ ! -z "${CIRCLE_TAG}" ]; then
                 # deploy a release tag
                 echo "${DOCKERHUB_REPO}:${CIRCLE_TAG}"
-                docker tag app:build "${DOCKERHUB_REPO}:${CIRCLE_TAG}"
+                docker tag "${LOCAL_IMAGE}" "${DOCKERHUB_REPO}:${CIRCLE_TAG}"
```

Comment:
We don't have an "app:build" in socorro, so this would have failed when tagging and pushing to prod. :(

I switched it to use `LOCAL_IMAGE` to make it more copy/paste friendly.